src/functions_linearalgebra.jl

```diff
@@ -118,3 +118,38 @@ function Base.getproperty(
         return inner
     end
 end
+
+function LinearAlgebra.:\(
+    fact::NamedFactorization{L, T, F},
+    nda::NamedDimsArray{W},
+) where {L, T, F<:Factorization{T}, W}
+    n1, n2 = L
+    n1 != only(W) && throw(ArgumentError("Dimension mismatch with dimensions $L and $W"))
+    return NamedDimsArray{(n2,)}(LinearAlgebra.:\(parent(fact), parent(nda)))
+end
+
+function LinearAlgebra.:\(
+    fact::NamedFactorization{L, T, F},
+    nda::AbstractVector,
+) where {L, T, F<:Factorization{T}}
+    n1, n2 = L
+    return NamedDimsArray{(n2,)}(LinearAlgebra.:\(parent(fact), nda))
+end
+
+# Specialised routines for \ often do in-place ops that result in the nameddim populated from B
+# Leading to an incorrect named-dim
+for S in (UpperTriangular, LowerTriangular)
```

Comment:
These got special-case because they ended up dispatching into [triangular ](https://github.com/JuliaLang/julia/blob/master/stdlib/LinearAlgebra/src/triangular.jl#L1641)